ModelViewer/ModelViewer/GraphicsWindowDelegate.h

```diff
@@ -1,4 +1,5 @@
 #pragma once
+#include "LandingPage.h"
```

Comment:
If you only need a pointer (or reference) to a class in a header, a good practice is to forward declare the class:
`class LandingPage;`

Then, the #include statement is only needed in the .cpp file.  This helps prevent circular dependencies.  In this case, there's no issue but just thought I'd point it out :)

Reply:
TIL :)

Reply:
Yay! That was the goal :)